### PR TITLE
Update articles

### PR DESCRIPTION
**Pull request by shushugah**
I improved the design of news index, individual news layout and the home page to preview 3 latest articles. In the case of IG Metal posts, I included canonical_url, to point to original article. 

@mleimeister and @JMCQ87 I'd love your feedback as well, since you both contributed articles. Does this make sense?

**Before and after**Before

And after

### CODE REVIEW

File: _data/authors.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+timo_daum:
```
Comment (chobeat) [inline _data/authors.yml:1]: maybe prepare already some default TWC author?

Comment (shushugah) [reply _data/authors.yml:1]: Agreed. It would be TechWorkersBER then, what should "name" be? "Berlin Tech Workers Coalition?"

**Comment by JMCQ87** [comment]
The icon on the two IG Metal articles does not make me think of TWC Berlin, but maybe rather Unicorns in Tech... which is maybe not a bad thing though. ;)

**Comment by mleimeister** [comment]
I think it looks great, and will ease the process of adding an article a lot :-)

File: _data/authors.yml (hunk 1 above)
Comment (chobeat) [reply _data/authors.yml:1]: yes, it's fine

**Comment by chobeat** [comment]
what is missing to merge this?

**Comment by shushugah** [comment]
Added default author of TechWorkersBer if none provided, see here if Timo_Daum was removed this would be the default author

Screenshot of posts, with Berlin TWC logo/name for an article without its own author

**Pull request merged**